template.go

```diff
@@ -75,16 +87,15 @@ func (r *{{$typename}}) Scan(i interface{}) error {
 	case []byte:
 		return r.UnmarshalJSON(t)
```

Comment:
Should this UnmarshalJSON call also be a setValue call instead now? Casting the bytes to a string